azurerm/resource_arm_mssql_server_security_alert_policy_test.go

```diff
@@ -0,0 +1,157 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMMssqlServerSecurityAlertPolicy_basic(t *testing.T) {
```

Comment:
Same here could we have both a basic and update test?

Reply:
done